eureka-client-archaius2/src/main/java/com/netflix/discovery/Ec2EurekaArchaius2InstanceConfig.java

```diff
@@ -0,0 +1,112 @@
+package com.netflix.discovery;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.netflix.appinfo.AmazonInfo;
+import com.netflix.appinfo.AmazonInfo.MetaDataKey;
+import com.netflix.appinfo.DataCenterInfo;
+import com.netflix.appinfo.DataCenterInfo.Name;
+import com.netflix.archaius.Config;
+
+/**
+ * When running in EC2 add the following override binding.
+ * 
+ * 	bind(EurekaInstanceConfig.class).to(KaryonEc2EurekaInstanceConfig.class);
+ * 
+ * 
+ * @author elandau
+ *
+ */
+@Singleton
+public class Ec2EurekaArchaius2InstanceConfig extends EurekaArchaius2InstanceConfig {
+    private static final Logger LOG = LoggerFactory.getLogger(Ec2EurekaArchaius2InstanceConfig.class);
+    
+    private static final String DEFAULT_NAMESPACE = "eureka";
```

Comment:
move this to the super class and then both can reference it instead of having a magic "eureka" in EurekaArchaius2InstanceConfig?